cmd/utils.go

```diff
@@ -60,3 +64,70 @@ func scanForProjectFiles(projType projectType) ([]string, error) {
 	}
 	return paths, nil
 }
+
+// findProject scans the directory for Xcode Project (.xcworkspace / .xcodeproject) file first
+// If can't find any, ask the user to drag-and-drop the file
+func findXcodeProject() (string, error) {
+	var projpth string
+
+	projPaths, err := scanForProjectFiles(iOSProjectType)
+	if err != nil {
+		log.Printf("Failed: %s", err)
+		fmt.Println()
+
+		log.Infof("Provide the project file manually")
+		askText := `Please drag-and-drop your Xcode Project (` + colorstring.Green(".xcodeproj") + `) or Workspace (` + colorstring.Green(".xcworkspace") + `) file, 
+the one you usually open in Xcode, then hit Enter.
+(Note: if you have a Workspace file you should most likely use that)`
+		projpth, err = goinp.AskForPath(askText)
+		if err != nil {
+			return "", fmt.Errorf("failed to read input: %s", err)
+		}
+		return projpth, err
```

Comment:
why do you return the error? i would explicitly highlight that in this case error is nil `return projpth, nil`

Reply:
fixed in:
https://github.com/bitrise-tools/codesigndoc/pull/97/commits/f16a378f594099e2f9d1fbcf54822c70d6e2e3ad